src/app/components/timer/Timer.ts

```diff
@@ -0,0 +1,32 @@
+export class Timer {
+    gameTimeInSec: number;
+    addedTimeInSec: number;
+    element: HTMLElement;
+
+    constructor(gameTimeInSec: number, addedTimeInSec: number) {
+        this.element = document.createElement('div');
+        this.gameTimeInSec = gameTimeInSec;
+        this.addedTimeInSec = addedTimeInSec;
+    }
+
+    startTime() {
+        const actualTimer = setInterval(() => {
+            this.gameTimeInSec--;
```

Comment:
Zastanów się co ma się stać gdy `this.gameTimeInSec` będzie mniejsze równe 0 .

Reply:
myślałem, żeby wywołać cb